models/quality_measures/intermediate/adh-ras_medication_adherence_for_hypertension/quality_measures__int_adhras_numerator.sql

```diff
@@ -0,0 +1,362 @@
+{{ config(
+     enabled = var('quality_measures_enabled',var('claims_enabled',var('clinical_enabled',var('tuva_marts_enabled',False)))) | as_bool
+   )
+}}
+
+with denominator as (
+
+    select
+          patient_id
+        , dispensing_date
+        , first_dispensing_date
+        , days_supply
+        , ndc_code
+        , performance_period_begin
+        , performance_period_end
+        , measure_id
+        , measure_name
+        , measure_version
+    from {{ ref('quality_measures__int_adhras_denominator')}}
+
+)
+
+, performance_end as (
+
+    select
+      performance_period_end
+    from {{ ref('quality_measures__int_adhras__performance_period') }}
+
+)
+
+/*
+The below 3 cte identifies periods of continuous medication use for each patient by:
+1. Assigning a row number and tracking the previous medication per patient.
+2. Flagging when a medication change occurs.
+3. Grouping consecutive periods of the same medication by assigning a group ID.
+*/
+
+, ranked_patient as (
+
+    select
+          patient_id
+        , dispensing_date
+        , ndc_code
+        , days_supply
+        , dense_rank() over (partition by patient_id order by dispensing_date) as row_number
```

Comment:
Update row_number alias to dense_rank wherever referenced as well
```suggestion
        , dense_rank() over (partition by patient_id order by dispensing_date) as dense_rank
```